integration_testing/features/learner/learner-engage-slideshow.feature

```diff
@@ -0,0 +1,59 @@
+Feature: Learner engages with content of the slideshow kind
```

Comment:
@radinamatic Would you be able to give this gherkin story a look and let me know if it covers everything you think that should be covered in a story for the slideshow?